Dockerfile

```diff
@@ -23,17 +23,26 @@ COPY main.go main.go
 COPY apis/ apis/
 COPY controllers/ controllers/
 COPY resources/ resources/
+COPY must-gather/ must-gather/
 
 # Build
 RUN VERSION=${BUILD_VERSION} GIT_COMMIT=${GIT_COMMIT} GIT_VERSION=${GIT_VERSION} make build
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
```

Comment:
I'm proposing `ubi8/ubi-micro` as a replacement for `distroless` as a base image for the operator's manager library, so that `/bin/bash` is available to run the must-gather collection script. 

See https://www.redhat.com/en/blog/why-distroless-containers-arent-security-solution-you-think-they-are

Reply:
This is an upstream Dockerfile. For our build we use this Dockerfile:
https://gitlab.cee.redhat.com/custom-metrics-autoscaler/cpaas-midstream/-/blob/ocma-2-rhel-8/distgit/containers/custom-metrics-autoscaler-operator/Dockerfile.in